setup-commit-signing/action.yml

```diff
@@ -0,0 +1,21 @@
+name: setup-commit-signing
+description: Set up GPG commit signing
+author: nandahkrishna
+branding:
+  icon: user-check
+  color: green
+inputs:
+  signing_key:
+    description: Secret key to use for signing commits
+    required: true
+  passphrase:
+    description: Secret key passphrase
+    required: false
```

Comment:
How does this work if it's not present (or is it just that secret keys technically don't require passphrases)?

Reply:
Keys don't require a passphrase, although it is recommended to have one.

Reply:
I think we should probably require it in this workflow because not setting one seems like A Very Bad Idea.